.github/workflows/r-ci.yml

```diff
@@ -72,6 +72,10 @@ jobs:
       - name: Update Packages
         run: Rscript -e 'update.packages(ask=FALSE)'
 
+      - name: Install SeuratObject from source
+        run: install.packages("SeuratObject", type = "source", repos = "https://cloud.r-project.org")
+        shell: Rscript {0}
```

Comment:
Please condition this on macOS only by adding a line such as

```
        if: ${{ matrix.os != 'macOS-latest' }}
```

which you see in other blocks in the file.  As mentioned over DM on slack, you may also need to take care of the build dependencies for SeuratObject.

r2u should supply us with a suitable update binary I injected earlier.

Reply:
Ah dang so it was my fault copy and pasting the wrong (non-macOS) line.  Doh.

Well we're getting there now with the rebuilt r2u binary...